.github/workflows/pull-request-management.yml

```
@@ -374,9 +447,18 @@ jobs:
             3.11
             3.12
             3.13
-      - name: "Install Python requirements"
+      - name: Download pyavd Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: pyavd
+          path: /tmp/
+      - name: Extract pyavd wheel package
+        run: tar -xzf /tmp/pyavd.tar.gz -C /tmp/
+      - name: "Install pyavd and Ansible requirements"
         run: |
-          pip install ${{ matrix.pip_installation }} -r .github/requirements-ci-dev.txt --upgrade
+          WHEEL_FILE=$(find /tmp/pyavd/ -name "*.whl")
+          pip install --no-cache-dir "$WHEEL_FILE[ansible-collection]"
+          pip install ${{ matrix.pip_installation }} --upgrade
```

```suggestion
          pip install --no-cache-dir "$WHEEL_FILE[ansible-collection]"
          pip install ${{ matrix.pip_installation }} --upgrade
```

why do you need two lines?

---

fixed